package.json

```diff
@@ -2,21 +2,26 @@
   "name": "@gnosis.pm/dex-js",
   "version": "0.0.1",
   "description": "",
+  "license": "MIT",
   "scripts": {
-    "sandbox": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config -r tsconfig-paths/register",
+    "build": "rimraf build && tsc",
     "test": "jest",
     "test-coverage": "jest --coverage && cat ./coverage/lcov.info | coveralls",
     "lint": "eslint src --ext .ts,.js",
     "lint:fix": "eslint --fix . --ext .ts,.js",
     "verify": "npm run lint && npm run test",
-    "build": "rimraf build && tsc",
-    "rpc": "ganache-cli -d --mnemonic 'candy maple cake sugar pudding cream honey rich smooth crumble sweet treat'"
+    "sandbox": "DEBUG=ERROR-*,WARN-*,INFO-*,DEBUG-*,-DEBUG-helper*,-DEBUG-util* nodemon -r dotenv/config -r tsconfig-paths/register"
   },
   "husky": {
     "hooks": {
       "pre-commit": "npm run verify"
     }
   },
+  "main": "build/src/index.js",
+  "files": [
+    "src",
```

Comment:
Any reason to publish `src` to npm?
There are valid reasons, but what are ours?

Reply:
I thought it would both document and allow others to use the originals TS if they please, not big reason behind this, I assume it doesn't harm

Anyways, now we talk about it, I'll go from a basic approach to add it only if we need it, or someone else needs it.